modules/apps/object/object-web/src/main/resources/META-INF/resources/js/tests/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeObjectFields.spec.tsx

```diff
@@ -0,0 +1,165 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {render} from '@testing-library/react';
+import React from 'react';
+
+import {ObjectDefinitionNodeFields} from '../../../components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeObjectFields';
```

Comment:
just to make sure, does the `ObjectDefinitionNodeObjectFields` name really need to have the word 'object' before fields? i'm asking because the name of the function is `ObjectDefinitionNodeFields` but the file is `ObjectDefinitionNodeObjectFields`

Reply:
Hey @rebsilva, I didn't notice this, I think it was an error in the renaming we did in Model Builder, I don't think we need the word object before the fields, so I think it would be better to rename the file... What do you think?

Great catch by the way!

Reply:
i think is better to rename it! but if you prefer to do this in another PR is fine by me

Reply:
Hey @rebsilva, I've renamed the tsx and the scss file :smile: